stwcs/wcsutil/altwcs.py

```diff
@@ -713,8 +749,7 @@ def getKeyFromName(header, wcsname):
     names = wcsnames(header)
     wkeys = [key for key, name in names.items() if name.lower() == wcsname.lower()]
     if wkeys:
-        wkeys.sort()
-        wkey = wkeys[-1]
+        wkey = max(wkeys)
```

Comment:
This kind of logic (`[ build a list of keys/names][-1]`) was already used in a couple of places and it is affected by uniqueness of the wcsnames. Therefore, it would be better to enforce uniqueness of at least Alt WCS names.

Reply:
I agree